### PR TITLE
Update issue templates to hide some template information

### PR DESCRIPTION
**Pull request by backwind1233**
Add comment tag to hide some template information when user create issues from templates.

- [bug_report_template](https://github.com/Azure-Samples/azure-spring-boot-samples/blob/5935534493813fcc48be0ea08010900199aae52f/.github/ISSUE_TEMPLATE/bug_report.md)
- [feature_request_template](https://github.com/Azure-Samples/azure-spring-boot-samples/blob/5935534493813fcc48be0ea08010900199aae52f/.github/ISSUE_TEMPLATE/feature_request.md)
- [question-query-template](https://github.com/Azure-Samples/azure-spring-boot-samples/blob/5935534493813fcc48be0ea08010900199aae52f/.github/ISSUE_TEMPLATE/question-query-template.md)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -8,40 +8,43 @@ assignees: ''
...
 
 **Setup (please complete the following information):**
- - OS: [e.g. iOS]
- - IDE: [e.g. IntelliJ]
- - Library/Libraries: [e.g. com.azure.spring:spring-cloud-azure-starter-keyvault-secrets:4.0.0 (groupId:artifactId:version)]
- - Java version: [e.g. 8] 
- - App Server/Environment: [e.g. Tomcat, WildFly, Azure Function, Apache Spark, Databricks, IDE plugin or anything special]
- - Frameworks: [e.g. Spring Boot, Micronaut, Quarkus, etc] 
-
+- Github link with tag or branch: <!-- [e.g. https://github.com/Azure-Samples/azure-spring-boot-samples/tree/spring-cloud-azure_v4.0.0/cosmos/azure-spring-data-cosmos/cosmos-multi-database-single-account] -->
```
Comment (saragluna) [inline .github/ISSUE_TEMPLATE/bug_report.md:29]: How about changing `Github link with tag or branch` to `Sample Path`

File: .github/ISSUE_TEMPLATE/question-query-template.md [hunk 2]
```diff
@@ -8,17 +8,19 @@ assignees: ''
...
+<!-- A clear explanation of why is this not a bug or a feature request? -->
 
 **Setup (please complete the following information if applicable):**
- - OS: [e.g. iOS]
- - IDE: [e.g. IntelliJ]
- - Library/Libraries: [e.g. com.azure.spring:spring-cloud-azure-starter-keyvault-secrets:4.0.0 (groupId:artifactId:version)]
- 
- **Information Checklist**
- Kindly make sure that you have added all the following information above and checkoff the required fields otherwise we will treat the issuer as an incomplete report
+- Github link with tag or branch: <!-- [e.g. https://github.com/Azure-Samples/azure-spring-boot-samples/tree/spring-cloud-azure_v4.0.0/cosmos/azure-spring-data-cosmos/cosmos-multi-database-single-account] -->
```
Comment (saragluna) [inline .github/ISSUE_TEMPLATE/question-query-template.md:17]: same here

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -8,40 +8,43 @@ assignees: ''
...
+- OS: <!-- [e.g. iOS] -->
+- IDE: <!-- [e.g. IntelliJ] -->
+- Library/Libraries: <!-- [e.g. com.azure.spring:spring-cloud-azure-starter-keyvault-secrets:4.0.0 (groupId:artifactId:version)] -->
```
Comment (saragluna) [inline .github/ISSUE_TEMPLATE/bug_report.md:32]: how about putting sample and library as close as possible:
- OS
- IDE
- Sample Path
- Library/Libraries

**Review by saragluna — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (backwind1233) [reply .github/ISSUE_TEMPLATE/bug_report.md:32]: I should have updated this order.

**Review by saragluna — APPROVED** [review]
LGTM.

**Pull request merged**